src/pages/PeoplePage.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { useEffect, useState } from 'react';
+import { Loader } from '../components/Loader';
+import { Person } from '../types';
+import { getPeople } from '../api';
+import { PeopleTable } from '../components/PeopleTable';
+import { findPeopleFamily } from '../helpers/findPeopleFamilly';
+
+export const PeoplePage: React.FC = () => {
+  const [peoples, setPeoples] = useState<Person[]>([]);
+  const [peopleLoading, setPeopleLoading] = useState(false);
+  const [peopleError, setPeopleError] = useState('');
+
+  useEffect(() => {
+    setPeopleLoading(true);
+    getPeople()
+      .then((peopleFromServer) => setPeoples(peopleFromServer))
+      .catch(() => {
+        setPeopleError('Something went wrong');
+      })
+      .finally(() => setPeopleLoading(false));
+  }, []);
+
+  const normalizePeoples = findPeopleFamily(peoples);
```

Comment:
You are running this on every render, do this in the useEffect above, or in useMemo